index.html

```diff
@@ -1,10 +1,10 @@
-<!--
+<!-- 
   generatePageContent.tsx looks for the following strings in this file:
   - "TITLE PLACEHOLDER"
   - "<style></style>"
   - "CONTENT PLACEHOLDER"
-  If you change any of these, make sure to edit generatePageContent.tsx as well.
- -->
+  If you change any of these, make sure to edit generatePageContent.tsx as well. -->
+
```

Comment:
this looks like junk, I would just revert this

Reply:
Agree, will update.